frontend/catalyst/compiler.py

```diff
@@ -117,27 +117,21 @@ def get_default_flags(options):
         # Discover the LAPACK library provided by scipy & add link against it.
         # Doing this here ensures we will always have the correct library name.
 
-        if platform.system() == "Linux":
-            file_path_within_package = "../scipy.libs/"
-            file_extension = ".so"
-        else:  # pragma: nocover
-            msg = "Attempting to use catalyst on an unsupported system"
-            assert platform.system() == "Darwin", msg
-            file_path_within_package = ".dylibs/"
-            file_extension = ".dylib"
-
-        package_name = "scipy"
+        package_name = "scipy_openblas32"
+        file_path_within_package = "lib"
+        file_extension = ".so" if platform.system() == "Linux" else ".dylib"
+
         scipy_package = importlib.util.find_spec(package_name)
         package_directory = path.dirname(scipy_package.origin)
         scipy_lib_path = path.join(package_directory, file_path_within_package)
```

Comment:
[Optional] Would it be worth renaming these variables to, e.g. `scipy_openblas32_package`, `scipy_openblas32_lib_path`, etc.? Otherwise it's a slight misnomer.

Reply:
Maybe something generic and simple like `openblas_package`, `openblas_package_directory` and `openblas_lib_path`? That way it's agnostic to where we're sourcing the OpenBLAS library from. Up to you :slightly_smiling_face:

Reply:
Looks good! :rocket: